AutoGenerateHeaderFiles.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+#  Created by Gershon Hochman on 18/01/2017.
+#  Copyright © 2017 Lightricks. All rights reserved.
+
+export ARCHS="i386 x86_64"
+export ARCHS_STANDARD="i386 x86_64"
+export ARCHS_STANDARD_32_64_BIT="i386 x86_64"
+export ARCHS_STANDARD_32_BIT=i386
+export ARCHS_STANDARD_64_BIT=x86_64
+export ARCHS_STANDARD_INCLUDING_64_BIT="i386 x86_64"
+export CURRENT_ARCH=x86_64
+export DEPLOYMENT_TARGET_CLANG_ENV_NAME=MACOSX_DEPLOYMENT_TARGET
+export DEPLOYMENT_TARGET_CLANG_FLAG_NAME=mmacosx-version-min
+export DEPLOYMENT_TARGET_SETTING_NAME=MACOSX_DEPLOYMENT_TARGET
+export EFFECTIVE_PLATFORM_NAME=-macosx
+unset IPHONEOS_DEPLOYMENT_TARGET
+export MACOSX_DEPLOYMENT_TARGET=10.12
+export NATIVE_ARCH=x86_64
+export PLATFORM_DEVELOPER_APPLICATIONS_DIR=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Applications
+export PLATFORM_DEVELOPER_BIN_DIR=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/usr/bin
+export PLATFORM_DEVELOPER_SDK_DIR=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs
+export PLATFORM_DEVELOPER_TOOLS_DIR=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Tools
+export PLATFORM_DEVELOPER_USR_DIR=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/usr
+export PLATFORM_DIR=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform
+export PLATFORM_DISPLAY_NAME=MacOSX
+export PLATFORM_NAME=macosx
+export PLATFORM_PREFERRED_ARCH=x86_64
+export SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk
```

Comment:
I prefer if Xcode can provide these, since you'll need to change `10.12` everywhere if you'll be using a different Xcode.